starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,36 @@
+
+
+.body {
+    font-family: sans-serif;
+    font-size : 20px;
+}
+.container {
```

Comment:
Here you placed your image inside a container and tried to position it inside, but as you saw the image is distorted and the placement isn't necessarily easy.
There's a better way to placing such big images, and it's the ``background-image`` CSS property. You basically create a ``container``div, give it dimensions and place it on the page, and then give it an image as a background.
(you can read more about it here: https://developer.mozilla.org/fr/docs/Web/CSS/background-image)

So here, you code could have looked like this:
```css
.container {
    position: relative;
    width: 85%;
    height: 500px;
    margin: 0 auto; /*centers your bloc horizontally*/
    background-image: url( "./../images/apple-pie.jpg" );
    background-size: cover; /*tells your image to take as much place as possible, without being distorted*/
    background-position: center; /*tells your image to be centered inside the bloc*/
}
```